src/pages/createjd.tsx

```diff
@@ -18,26 +18,31 @@ const JdForm: NextPage = () => {
     formState: { errors },
   } = useForm<IJdForm>();
 
-  const onSubmit: SubmitHandler<IJdForm> = (data) => {
+  useEffect(() => {
     const tempValue = localStorage.getItem('jobLisId');
     if (typeof tempValue === 'string') {
       setValue(JSON.parse(tempValue));
     }
+  }, []);
```

Comment:
I noticed that you were using local storage to get the `jobListingId`.

This can make it confusing to work with because it's not immediately obvious how the jobListingId gets passed to the `CreateJobDescription` Form.

Because of that, I think we should take a different approach to localStorage.

Instead, what are your thoughts on passing the `jobListingId` as part the the URL params?
``localhost:3000/createjd/?jobListingId=${jobListingId}``

After that, we can use the Next.js `router.query` prop to retrieve the value. 

Relevant docs: https://nextjs.org/docs/api-reference/next/router

**For now, let's look at this in another GitHub issue/ticket. I missed this last time and it's already on the main branch.**

Reply:
I found a bug when you jump to the CreateJobDescription page. It uses a stale value for the JobListingId in localStorage.

![image](https://user-images.githubusercontent.com/8443215/221137734-e89f2489-b14d-44a9-9564-0fb93ba7e729.png)